core/config.go

```diff
@@ -0,0 +1,63 @@
+package core
+
+import (
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"time"
+)
+
+type Config struct {
+	Pebble struct {
+		ListenAddress   string
+		HTTPPort        int
+		TLSPort         int
+		ValidationSleep ConfigDuration
```

Comment:
I was actually thinking more along the lines of an environment variable, so it would be easy to override without creating diffs to the config that we'd have to remember not to commit. Sorry I forgot to mention this yesterday; I think we had talked about the possibility, but it was a long time ago.